pegasus/migrations/76_migrate_channel_sources_to_s3.rb

```diff
@@ -0,0 +1,60 @@
+require 'cdo/hip_chat'
+require 'files_api'
+
+# Migration for moving JSON.parse(row[:value])['levelSource'] and ['levelHtml'] to S3 (/v3/sources API).
+Sequel.migration do
+  up do
+    HipChat.log 'Moving <b>channel</b> sources to S3...'
+    FILE_NAME = 'main.json'
+
+    # For each channel with a level(Source|Html) and no migratedToS3 flag, create a new S3 record and set migratedToS3.
+    batch_update do |row|
+      begin
+        value = JSON.parse(row[:value])
+      rescue JSON::ParserError
+        next
```

Comment:
:+1: we shouldn't have any channels which became unparsable recently because the original bug was fixed, right? As long as they're all at least 15 days old I'd be ok with deleting all of them.